staging/src/k8s.io/client-go/transport/round_trippers.go

```diff
@@ -61,9 +65,27 @@ func HTTPWrappersForConfig(config *Config, rt http.RoundTripper) (http.RoundTrip
 		len(config.Impersonate.Extra) > 0 {
 		rt = NewImpersonatingRoundTripper(config.Impersonate, rt)
 	}
+	rt = NewOtelRoundTripper(rt)
 	return rt, nil
 }
 
+type otelWrapper struct {
+	otelRT http.RoundTripper
+	baseRT http.RoundTripper
+}
+
+// NewOtelRoundTripper returns an otelhttp-wrapped round tripper that implements
+// WrappedRoundTripper
+func NewOtelRoundTripper(rt http.RoundTripper) http.RoundTripper {
+	return &otelWrapper{otelRT: otelhttp.NewTransport(rt, otelhttp.WithPropagators(otel.NewCompositeTextMapPropagator(propagators.TraceContext{}, propagators.Baggage{}))), baseRT: rt}
```

Comment:
note to reviewers: This is a no-op unless a trace "sdk" is registered.  See the component base utils for how that can be done.